inventory/byo/hosts.origin.example

```diff
@@ -536,9 +536,9 @@ openshift_master_identity_providers=[{'name': 'htpasswd_auth', 'login': 'true',
 # this value must be 1
 #openshift_hosted_logging_elasticsearch_cluster_size=1
 #openshift_hosted_logging_hostname=logging.apps.example.com
-# Configure the prefix and version for the deployer image
-#openshift_hosted_logging_deployer_prefix=registry.example.com:8888/openshift3/
-#openshift_hosted_logging_deployer_version=3.3.0
+# Configure the prefix and version for the component images
+#openshift_hosted_logging_deployer_prefix=docker.io/openshift/origin-
```

Comment:
@sdodson do we want to use the correct prefix/versions for origin? or should it still match the previous examples (but bump to 3.5.0)